cmd/cqlplay/main.go

```diff
@@ -107,7 +108,11 @@ func handleEvalCQL(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	elm, err := cql.Parse(req.Context(), []string{evalCQLReq.CQL, fhirHelpers}, cql.ParseConfig{DataModels: [][]byte{fhirDM}})
+	// Combine main CQL with additional libraries and FHIRHelpers
+	cqlInputs := append([]string{evalCQLReq.CQL}, evalCQLReq.Libraries...)
+	cqlInputs = append(cqlInputs, fhirHelpers)
```

Comment:
we may not want to always add fhirHelpers, and/or try to detect if the user has added a version of fhirHelpers already